go/chat/sync.go

```diff
@@ -228,18 +228,34 @@ func (s *Syncer) handleFilteredConvs(ctx context.Context, uid gregor1.UID, syncC
 	}
 }
 
+func (s *Syncer) maxSyncUnboxConvs() int {
+	if s.G().IsMobileAppType() {
+		return 8
+	}
+	return 100
+}
+
 func (s *Syncer) getShouldUnboxSyncConvMap(ctx context.Context, convs []chat1.Conversation,
 	topicNameChanged []chat1.ConversationID) (m map[string]bool) {
 	m = make(map[string]bool)
 	for _, t := range topicNameChanged {
 		m[t.String()] = true
 	}
-	for _, conv := range convs {
-		if m[conv.GetConvID().String()] {
+	rconvs := utils.RemoteConvs(convs)
+	sort.Slice(rconvs, func(i, j int) bool {
```

Comment:
can you move this into utils so when we change the priority metric to unreadline (or something) we can change everything in one place?

Reply:
I want this in time order, the point is to only unbox the stuff on the screen.